src/CarbonAware.WebApi/src/Program.cs

```diff
@@ -56,4 +56,6 @@
 
 app.MapHealthChecks("/health");
 
-app.Run();
\ No newline at end of file
+app.Run();
+
+public partial class Program { }
```

Comment:
Is this needed?

Reply:
@KanishkT123 I think we need this for WebApplicationFactory correct?

Reply:
I'll add a comment, but this is documented and needed, yep.